doc/source/user_guide_embedding/libraries.rst

```diff
@@ -0,0 +1,34 @@
+.. _ref_embedding_user_guide_libraries:
+
+*********
+Libraries
+*********
+
+
+.. note::
+
+    This is an experimental feature. Some of these libraries will not work.
+
+Most of Mechanical's scripting APIs are implemented in C#. However, there are a small number
```

Comment:
These modules are not available for use from an embedded instance of Mechanical in python because Python does not know where to find those.
But, in order to use these modules, you need to use the function ``add_mechanical_python_libraries`` to find and import certain modules into Python. 
In addition, it is necessary to first initialize the embedded instance of Mechanical because certain libraries expect the .NET Common Language Runtime to be initialized as well as for the appropriate C# libraries to be loaded.
To use the above function, run the following: